### PR TITLE
[MI-1995] Fixed failing CI in link tooltip PR

### PR DESCRIPTION
**Pull request by raghavaggarwal2308**
1. Fixed the error "'gitlab.com' can be anywhere in the URL, and arbitrary hosts may come before or after it."
Screenshot from 2022-07-28 15-36-01

### CODE REVIEW

**Review by sooraj-shukla — APPROVED** [review]

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ on:
     branches: [ master ]
   pull_request:
     # The branches below must be a subset of the branches above
-    branches: [ master ]
+    branches: [ master, link_tooltip ]
```
Comment (manojmalik20) [inline .github/workflows/codeql-analysis.yml:8]: This change was only for testing if the fix worked, right? You should revert this change now.

**Review by manojmalik20 — APPROVED** [review]
